docs/config/front-matter.md

```diff
@@ -31,6 +53,10 @@ date: 2016-10-20
 
 ## author
 
+- Type: `string`
```

Comment:
Not critical to this PR, but what do you think about making Author an object instead so that people can define things like meta information (i.e., social media links, email, etc.)

Reply:
What do you recommend to leverage those meta information? Currently, people can display contact infos by the option `footer.contact`.